src/database/versions.ts

```diff
@@ -26,16 +26,14 @@ export const dbVersions = {
   }
 };
 
-export const dbHistory = ['v1.0.0', 'v0.0.1'];
+export const dbHistory = ['v1.0.0', 'v0.0.1'] as const;
 
-// @ts-ignore
 export const getCurrentDBConfig = () => dbVersions[dbHistory[0]];
 
-// @ts-ignore
 export const getPreviousDBConfig = () => dbVersions[dbHistory[1]];
 
-export const getExportFileName = (currentDb: DBConfig, m: moment.Moment) => {
-  return `${SETTINGS_FILENAME}_${m.format('YYYY-MM-DD')}_${currentDb.version}.json`;
+export const getExportFileName = (currentDb: DBConfig, date: Date) => {
+  return `${SETTINGS_FILENAME}_${format(date, 'yyyy-MM-dd')}_${currentDb.version}.json`;
```

Comment:
Can you use only uppercase or lowercase ?

Reply:
`date-fns` uses `yyyy` for the full calendar year, `YYYY` is for the local week-numbering year. Not sure what is the difference is tbh. `dd` is for the day of month, `DD` for the day of year.

Reply:
> A common mistake is to use YYYY. yyyy specifies the calendar year whereas YYYY specifies the year (of “Week of Year”), used in the ISO year-week calendar. In most cases, yyyy and YYYY yield the same number, however they may be different. Typically you should use the calendar year.

So I'm assuming `yyyy` is correct.

Reply:
Thanks for the clarification ! 💯